libraries/ESP8266WebServer/examples/FSBrowser/FSBrowser.ino

```diff
@@ -257,54 +628,48 @@ void setup(void) {
   DBG_OUTPUT_PORT.print("Connected! IP address: ");
   DBG_OUTPUT_PORT.println(WiFi.localIP());
 
-  MDNS.begin(host);
-  DBG_OUTPUT_PORT.print("Open http://");
-  DBG_OUTPUT_PORT.print(host);
-  DBG_OUTPUT_PORT.println(".local/edit to see the file browser");
+  ////////////////////////////////
+  // MDNS INIT
+  if (MDNS.begin(host)) {
+    MDNS.addService("http", "tcp", 80);
+    DBG_OUTPUT_PORT.print("Open http://");
+    DBG_OUTPUT_PORT.print(host);
+    DBG_OUTPUT_PORT.println(".local/edit to open the FileSystem Browser");
+  }
+
+  ////////////////////////////////
+  // WEB SERVER INIT
 
+  // Filesystem status
+  server.on("/status", HTTP_GET, handleStatus);
 
-  //SERVER INIT
-  //list directory
+  // List directory
   server.on("/list", HTTP_GET, handleFileList);
-  //load editor
-  server.on("/edit", HTTP_GET, []() {
-    if (!handleFileRead("/edit.htm")) {
-      server.send(404, "text/plain", "FileNotFound");
-    }
-  });
-  //create file
-  server.on("/edit", HTTP_PUT, handleFileCreate);
-  //delete file
-  server.on("/edit", HTTP_DELETE, handleFileDelete);
-  //first callback is called after the request has ended with all parsed arguments
-  //second callback handles file uploads at that location
-  server.on("/edit", HTTP_POST, []() {
-    server.send(200, "text/plain", "");
-  }, handleFileUpload);
-
-  //called when the url is not defined here
-  //use it to load content from SPIFFS
-  server.onNotFound([]() {
-    if (!handleFileRead(server.uri())) {
-      server.send(404, "text/plain", "FileNotFound");
-    }
-  });
-
-  //get heap status, analog input value and all GPIO statuses in one json call
-  server.on("/all", HTTP_GET, []() {
-    String json('{');
-    json += "\"heap\":" + String(ESP.getFreeHeap());
-    json += ", \"analog\":" + String(analogRead(A0));
-    json += ", \"gpio\":" + String((uint32_t)(((GPI | GPO) & 0xFFFF) | ((GP16I & 0x01) << 16)));
-    json += "}";
-    server.send(200, "text/json", json);
-    json.clear();
-  });
```

Comment:
Why remove this?

Reply:
I thought it was a leftover from another example, because as far as I could see, it was unused by the html code, and had nothing to do with filesystem browsing. 
Now if you think it's of any use here, I can add it back of course...

Reply:
By the way, I also removed the data/graphs.js.gz file. Probably they were meant to work together, but no HTML code referred to that javascript file anymore.
Maybe they are valuable and could be used in a separate example, but we'd have to dig through the project history to find how they were meant to be used...

Reply:
They **are** supposed to work together, I have a variation of this working in my own webapp, where I read a bunch of values and show a bunch of graphs in a webpage using graphs.js.gz. Most likely the html page front end for /all got lost at some point in the near past.
In this PR I suggest:
- keeping this code, maybe rename /all to something like /data
- keeping graphs.js.gz, maybe see if there's some updated version of it

Then, if you have the time/inclination, do one of the following in a subsequent PR:
- implement the web front-end to show the graphs
or
- propose an alternative way of graphing data that is as self-contained as this one. This one works, but is rather limited.

Reply:
Hmm, I'm still tempted to think an example should do one thing and do it well, and this part clearly has nothing to do with filesystem browsing. 
But I agree that if there is valuable code that shows how to display data from the ESP in a web page (I guess), it shouldn't just get lost.
What about creating a dedicating example with that code ? 
I volunteer to do it once this PR is wrapped up...

Reply:
In this case I remember that the first time I tried the example I had a problem with SPIFFS self/destructing, so no files were present on boot no matter what I did. That meant that I would pretty much get a 404 everywhere. The only way I could know that the server was actually ok was thanks to this endpoint.
However, agreed, let's split into a specialized example for graphs. It's something that deserves an update anyways. In particular, I think the graph library should be replaced with something standard rather than an in-house lib.